sql/src/test/resources/model/model_file_csv.json

```diff
@@ -0,0 +1,27 @@
+{
+  version: '1.0',
```

Comment:
Nope.. That's calcite specific format for model file. Its not exactly json.. Its more of quasi json one.